x/vesting/keeper/integration_test.go

```diff
@@ -453,6 +453,64 @@ var _ = Describe("Clawback Vesting Accounts - claw back tokens", Ordered, func()
 		s.Require().Equal(balanceGrantee, bG)
 		s.Require().Equal(balanceDest, bD)
 	})
+
+	It("should update vesting funder and claw back unvested amount before cliff", func() {
+		ctx := sdk.WrapSDKContext(s.ctx)
+		newFunder := sdk.AccAddress(tests.GenerateAddress().Bytes())
+
+		balanceFunder := s.app.BankKeeper.GetBalance(s.ctx, funder, stakeDenom)
+		balanceNewFunder := s.app.BankKeeper.GetBalance(s.ctx, newFunder, stakeDenom)
+		balanceGrantee := s.app.BankKeeper.GetBalance(s.ctx, grantee, stakeDenom)
+
+		// Update clawback vesting account funder
+		updateFunderMsg := types.NewMsgUpdateVestingFunder(funder, newFunder, grantee)
+		_, err := s.app.VestingKeeper.UpdateVestingFunder(ctx, updateFunderMsg)
+		s.Require().NoError(err)
+
+		// Perform clawback before cliff - funds should go to new funder (no dest address defined)
+		msg := types.NewMsgClawback(newFunder, grantee, sdk.AccAddress([]byte{}))
+		_, err = s.app.VestingKeeper.Clawback(ctx, msg)
+		Expect(err).To(BeNil())
```

Comment:
Is there a specific reason for this not to use `s.Require().NoError(err)` as in the rest of the suite?